app/AppManifest.json

```diff
@@ -0,0 +1,45 @@
+[
+    {
+        "Name": "sampleapp",
+        "Port": -1,
+        "DAPR_GRPC_PORT": 50001,
+        "Dockerfile": "./app/Dockerfile",
+        "dependencies": {
+            "services": [
+                {
+                    "name": "seatservice",
+                    "image": "ghcr.io/eclipse/kuksa.val.services/seat_service",
+                    "version": "v0.1.0"
+                }
+            ],
+            "runtime": [
+                {
+                    "name": "mosquitto",
+                    "image": "eclipse-mosquitto",
+                    "version": "2.0.14"
+                },
+                {
+                    "name": "databroker",
+                    "image": "ghcr.io/eclipse/kuksa.val/databroker",
+                    "version": "v0.17.0"
+                },
+                {
+                    "name": "feedercan",
+                    "image": "ghcr.io/eclipse/kuksa.val.feeders/dbc2val",
+                    "version": "v0.1.0"
+                }
+            ],
+            "python": {
```

Comment:
Our sample app is cpp  application, is python should be part of the manifest?

Reply:
Good catch, removed.